src/typeguard/_checkers.py

```diff
@@ -782,7 +794,7 @@ def check_type_internal(
     IO: check_io,
     list: check_list,
     List: check_list,
-    Literal: check_literal,
+    typing_extensions.Literal: check_literal,
```

Comment:
If this is the only thing that needs `typing_extensions`, it could just be conditionally imported (`try...except ImportError`).